### PR TITLE
Fix cert-manager and iap config to make dashboard accessible

### PR DESCRIPTION
**Pull request by sambaiz**
This make centraldashboard accessible from Cloud Endpoint

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/1544)

### CODE REVIEW

**Comment by abhi-g** [comment]
Looks like the tests is failing due to format testing of cert-manager.libsonnet. Possibly due to the comment line. Try using /* C style comment */ or # python style comment.

File: kubeflow/core/cert-manager.libsonnet [hunk 1]
```diff
@@ -170,6 +170,8 @@
             name: "letsencrypt-prod-secret",
           },
           http01: {
+            // ksonnet removes an empty object
+            "dummy": "",
```
Comment (abhi-g) [inline kubeflow/core/cert-manager.libsonnet]: remove the " around dummy. "dummy" -> dummy. It should work after that.

**Comment by sambaiz** [comment]
Thanks!

**Comment by abhi-g** [comment]
@sambaiz Can you please merge upstream/master into the PR. I think otherwise its looking good.

**Comment by sambaiz** [comment]
@abhi-g I have merged. Can you please review?

File: kubeflow/core/cert-manager.libsonnet [hunk 2]
```diff
@@ -170,6 +170,8 @@
             name: "letsencrypt-prod-secret",
           },
           http01: {
+            /* ksonnet removes an empty object */
```
[the remark this answers is not in the document]
Comment (sambaiz) [reply kubeflow/core/cert-manager.libsonnet]: No we need not now. I have been removed it.

**Pull request merged**